pkg/controllers/dynakube/extension/eec/statefulset.go

```diff
@@ -47,6 +48,8 @@ const (
 	envEecHttpsPrivKeyPathPem   = httpsCertMountPath + "/" + consts.TLSKeyDataName
 	// Volume names and paths
 	eecTokenMountPath                  = "/var/lib/dynatrace/remotepluginmodule/secrets/tokens"
+	eecCustomCertificateMountPath      = "/var/lib/dynatrace/remotepluginmodule/secrets/extensions"
+	customCertificateVolumeName        = "extension-custom-certs"
```

Comment:
why does some const have an `eec` prefix in their name, while others don't ?
(especially considering that we are in the `eec` package)

Reply:
on it,.

Reply:
@0sewa0 fixed in 60ddd312